website/docs/reference/projects.md

```diff
@@ -3,131 +3,119 @@ id: projects
 
-A common pattern is to organize the feature flags according to key areas of the application, e.g. “Basic user process” and “Advanced user process”. This is illustrated below.
+You can configure which [environments](./environments.md) are available within a project. By default, all global environments are available. You can only enable and disable feature flags for the environments you've configured for the project.
 
-![A diagram with two boxes labeled "Basic user process" and "Advanced user process", respectively. The former contains features "New login" and "Winter theme enablement", the latter "New in-app purchase" and "Updated invoice repository".](/img/project_concept.png)
 
-## Projects and environments
+## Create a project
 
-You can configure which [environments](./environments.md) should be available within a project. By default, all globally available environments are available. You can only enable/disable a feature flag for the environments you configure for a project.
+:::note Availability
 
-Within the admin UI, the projects are available under the "environments" tab of the project page. Changing them requires the project owner role.
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing).
 
-## Creating a new project {#creating-a-new-project}
-
-When you log into Unleash for the first time, there is a Default project already created. All feature flags are included in the Default project, unless explicitly set to a different one.
+:::
 
-From the top-line menu – click on “Projects”
+To create a new project:
+1. Go to **Projects** > **New project** and enter a project name.
+2. Optionally configure settings such as [environments](/reference/environments.md), [stickiness](/reference/stickiness.md), [collaboration mode](/reference/project-collaboration-mode.md), and [change requests](/reference/change-requests.md).
+3. Click **Create project**.
 
-![The Unleash admin UI with the "Projects" nav link in the top bar highlighted.](/img/projects_button.png)
 
-The UI shows the currently available projects. To create a new project, use the “new project” button.
+## Update a project
 
-![A list of projects. There is a button saying "Add new project".](/img/projects_new_project.png)
+:::note Availability
 
-The configuration of a new Project is now available. the following input is available to create the new Project.
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing).
 
-![A project creation form. The "Create" button is highlighted.](/img/projects_save_new_project_v2.png)
+:::
 
-| Item               | Description                                                                                 |
-|--------------------|---------------------------------------------------------------------------------------------|
-| Project Id         | Id for this Project                                                                         |
```

Comment:
Added links in the instructions, otherwise the rest is self-explanatory in the interface.